CHANGELOG

```diff
@@ -83,6 +94,9 @@ v3.7.3 (XXXX-XX-XX)
   - `arangodb_http_request_statistics_user_requests`: Total number of HTTP
     requests executed by clients
 
+* Make scheduler react and start new threads slightly faster in case a lot
+  of new work arrives.
+
```

Comment:
I don't think this belongs here.

Reply:
this came with the merge, but will remove, if you say so. cannot comment, if it belongs there